src/main/java/com/checkmarx/flow/controller/ADOController.java

```diff
@@ -334,6 +352,20 @@ private String determineAzureProject(Repository repository) {
         return azureProject;
     }
 
+    private String getConfigBranch(ScanRequest request, Resource resource, Action action){
+        String branch = request.getBranch();
+        try{
+
+            if (isDeleteBranchEvent(resource) && action.equals(Action.PUSH)){
+                branch = request.getDefaultBranch();
+                log.debug("branch to read config-as-code: {}", branch);
+            }
+        }
+        catch (Exception ex){
+            log.error("failed to get branch for config as code. using default");
+        }
+        return branch;
+    }
```

Comment:
I'm not sure there is a scenario we can get into the catch clause since nothing in the 'try' throws an exception...

Reply:
If request default branch is null for example